v1.1/stop-a-node.md

```diff
@@ -14,6 +14,11 @@ For information about permanently removing nodes to downsize a cluster or react
 
 ### How It Works
 
+- Finishes in-flight requests.  Note that this is a best effort that times out after the duration specified by the `???` cluster setting (1.1.5)
```

Comment:
This setting is not in v1.1. The server just cancels all current sessions without waiting.

Reply:
Thanks, updated to say that in 14139e9b